test/jdk/java/awt/Dialog/FileDialogIconTest/FileDialogIconTest.java

```diff
@@ -38,7 +39,7 @@
 
 /*
  * @test
- * @bug 4035189
+ * @bug 6425126
  * @summary Test to verify that PIT File Dialog icon not matching with
  *          the new java icon (frame Icon) - PIT build
```

Comment:
I believe the File Dialog icon should match with the new java icon is correct behaviour. So, isn't it better to remove **not** from the summary?